Nos/Extensions/Font.swift

```diff
@@ -156,10 +162,10 @@ func Text(_ content: any StringProtocol) -> SwiftUI.Text {
 }
 
 func TextField(_ titleKey: LocalizedStringKey, text: Binding<String>, axis: Axis = .horizontal) -> some View {
-    SwiftUI.TextField(titleKey, text: text, axis: axis).font(.brand)
+    SwiftUI.TextField(titleKey, text: text, axis: axis).font(.body)
 }
 
 func TextField<Label: View>(text: Binding<String>, prompt: Text? = nil, label: () -> Label) -> some View {
-    SwiftUI.TextField(text: text, prompt: prompt?.font(.brand), label: label).font(.brand)
+    SwiftUI.TextField(text: text, prompt: prompt?.font(.brand), label: label).font(.body)
```

Comment:
What about getting rid of these as well? So we stop using the longer `SwiftUI.TextField` and `SwiftUI.Text` when we want a different font or a different constructor. IMO, it makes code look strange when it is not working as you intended and then you realize it's because we are "overriding" the native Text and TextField, and why we sometimes have a PlainText instead.

Reply:
@martindsq I'd _love_ to get rid of those but unfortunately that's outside the scope of this PR. If you want to create a ticket for getting rid of them, I'd appreciate it.

Reply:
@martindsq I actually just found that `TextField(text:prompt:label:)` is unused, so I'm deleting it. But there will still be some left over that we'll need to deal with in a future PR.

Reply:
We can do the same here, just marking them as deprecated so we don't forget about this.

Reply:
@martindsq I tried, but I can't bring myself to commit, push, and merge this:

<img width="448" alt="Screenshot 2024-03-14 at 4 20 28 PM" src="https://github.com/planetary-social/nos/assets/59564/5d06b4ec-7cce-4685-9ad0-9815bab2601a">

As you can see, there are a total of 172 issues when I mark things as deprecated, and that's just too much noise. I'm concerned we'll miss important warnings. I've created a new ticket to deal with this: https://github.com/planetary-social/nos/issues/955

Reply:
OK, sounds good